java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -316,6 +318,75 @@ public interface ServerManagementClusterCommands {
      */
     CompletableFuture<ClusterValue<Long>> lastsave(Route route);
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * The command will be routed to all primary nodes.
+     * <br.
+     * To explicitly specify the flushing mode, use {@link #flushall(FlushOption)}.
+     *
+     * @see <a href="https://redis.io/commands/flushall/">
+     * redis.io</a> for details.
```

Comment:
omg spotless...

Reply:
That time it is my fault, thank you for detecting that